README.md

```diff
@@ -1,24 +1,69 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
- [ ] In keeping with the Creative Commons License, in your README documentation, there should be an **Acknowledgment** section that gives due credit to [Gregoire Vella](https://www.behance.net/gregoirevella) the author of the design. Kindly add this.